CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 ### Added
+- Added key cache parsing for newer nodetool version
+
+### Added
```

Comment:
No need for this, they are both just items in an list of things added in the next release.